docs/src/references/mappers/c8y-mapper.md

```diff
@@ -568,6 +568,305 @@ c8y/alarm/alarms/create
 
 </div>
 
+### Data
+
+The data telemetry type can be used to store additional information about the device/child/service. Such information could included; operation system name/version, communication statistics, device status, or any other information that is not suited to measurements, events or alarms.
+
+#### Data - Main device
+
+A device's digital twin model can be updated by publishing to a specific topic.
+
+The type part of the topic is used to group the data so it is easier for components to subscribe to relevant parts.
+
+<div class="code-indent-left">
+
+**Thin Edge (input)**
+
+```text title="Topic (retain=true)"
+te/device/main///data/device_OS
```

Comment:
If *two* messages are sent for the *same* device, say `te/device/main///data/a` and `te/device/main///data/a`,
then which component is responsible of the combination of these two fragments?

In other words:
- does the mapper have to send a *single*  `{"a": ...,"b": ...}` message to `c8y/inventory/managedObjects/update/<main-device-id>` ?
- or can the mapper send a first `{"a": ...}` message then a second `{"b": ...}` message to C8y?

Reply:
Fortunately the mapper can send different fragments one after the other and they are aggregated at the cloud.

Reply:
> Fortunately the mapper can send different fragments one after the other and they are aggregated at the cloud. But, if some of the fields within a fragment needs to be updated, then the mapper needs to send the whole fragment with old values as well as new values. That's acceptable IMO.

We can optimize later to group the updates into one message